docker-compose.yml

```diff
@@ -7,3 +7,8 @@ services:
     environment:
       - SMARTY_AUTH_ID
       - SMARTY_AUTH_TOKEN
+      - SMARTY_URL_INTERNATIONAL
```

Comment:
To future-proof this variable, I recommend we use `SMARTY_URL_INTERNATIONAL_STREET` here.

Reply:
Updated.